src/_data/sidenav.yml

```diff
@@ -125,18 +125,22 @@
     - title: Platform integration
       permalink: /docs/development/platform-integration
       children:
-        - title: Adding Flutter to an Android app
-          permalink: /docs/development/platform-integration/add-to-app-android
+        - title: Add Flutter to existing app
+          permalink: /docs/development/platform-integration/add-to-app
           children:
-            - title: Add a single Flutter screen
-              permalink: /docs/development/platform-integration/add-to-app-android/add-flutter-screen
-            - title: Add a Flutter Fragment
-              permalink: /docs/development/platform-integration/add-to-app-android/add-flutter-fragment
-            - title: Add a Splash Screen
-              permalink: /docs/development/platform-integration/add-to-app-android/add-splash-screen
+          - title: Adding to an Android app
+            permalink: /docs/development/platform-integration/add-to-app/android
```

Comment:
This link seems to be broken in the staging website. Also, due to how low this has been moved, the individual Android options can no longer be seen in the side bar, which I think we probably should revert. I know that I'd certainly want to see the individual topics in the navigation area.

Reply:
ya, I noticed last night too. 

@Sfshaza2 should I try to tweak this in the implementation or is the depth intentionally fixed to this level?